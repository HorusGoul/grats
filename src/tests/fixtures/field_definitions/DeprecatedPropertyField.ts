/** @GQLType */
export default class Query {
  /**
   * @GQLField
   * @deprecated Use something else.
   */
  hello: string;
}
