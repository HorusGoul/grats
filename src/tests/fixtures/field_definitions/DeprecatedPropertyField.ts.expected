-----------------
INPUT
----------------- 
/** @GQLType */
export default class Query {
  /**
   * @GQLField
   * @deprecated Use something else.
   */
  hello: string;
}

-----------------
OUTPUT
-----------------
directive @renameField(name: String!) on FIELD_DEFINITION

type Query {
  hello: String @deprecated(reason: "Use something else.")
}